auth/jwt.go

```diff
@@ -0,0 +1,67 @@
+package auth
+
+import (
+	"github.com/dgrijalva/jwt-go"
+
+	"crypto/rsa"
+	"errors"
+	"fmt"
+	"time"
+)
+
+// todo: key rotation
+type Jwt struct {
```

Comment:
Please find a better name for this object. It is not a JWT per se. What is it?

Reply:
Fixed